src/component/restaurantTabButton.ts

```diff
@@ -0,0 +1,63 @@
+import { controlFavoriteIcon } from "../domain/favoriteIconController";
+import { updateRestaurants } from "../domain/filter";
+import { tabButtonType } from "../type";
+import { executeEventListener } from "../util/eventListener";
+import { $, $$ } from "../util/selector";
+
+// UI
+const renderTemplate = (buttonList: tabButtonType) => {
+  return `<button
+      id="tab-button${buttonList.key}"
+      class="text-body viewer-button ${buttonList.state ? "active" : ""}"
+    >
+      ${buttonList.name}
+    </button>`;
+};
```

Comment:
받아오는 인자의 종류가 변경 된 것 같아서, -List suffix는 빠져도 될 것 같아요.

```suggestion
const renderTemplate = (tabButton: tabButtonType) => {
	...
};
```